src/components/authentications/social/facebook/FacebookLogin.jsx

```diff
@@ -1,21 +1,55 @@
 import React, { Component } from 'react'
+import { Redirect } from 'react-router-dom'
+import axios from 'axios'
+import FacebookLogin from 'react-facebook-login'
 
-class FacebookLogin extends Component {
-	state = {}
+class FacebookLoginComponent extends Component {
+	state = {
+		status: ''
+	}
 
-	handleAuth = () => {
+	responseFacebook = async (res) => {
+		try {
+			console.log(res)
+			const data = await axios.post('http://192.168.1.120:3000/v1/auth/facebook', { accessToken: res.accessToken })
```

Comment:
create a variable so that when it is deployed in server it automatically takes url. @rubymaharjan